test/utils/layout-helpers-test.ts

```diff
@@ -37,11 +37,13 @@ describe('definition-helpers', function () {
     it('return package root folder', async function () {
       const resultOne = await resolvePackageRoot(__dirname, 'lodash');
 
-      expect(resultOne).toContain(path.sep + 'lodash');
+      // no lodash here.
+      expect(resultOne).toBe(false);
```

Comment:
it's incorrect, we do have `lodash` as dependency for ember-language server.

```sh
(base) lifeart@MacBookPro ember-language-server % yarn w
hy lodash
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "lodash"...?
[2/4] 🚚  Initialising dependency graph...
warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^6.0.1"
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "lodash@4.17.21"
info Has been hoisted to "lodash"
info Reasons this module exists
   - Specified in "dependencies"
   - Hoisted from "webpack-bundle-analyzer#lodash"
   - Hoisted from "@semantic-release#changelog#lodash"
   - Hoisted from "@semantic-release#git#lodash"
   - Hoisted from "ember-template-recast#async-promise-queue#async#lodash"
   - Hoisted from "jest#@jest#core#jest-config#jest-environment-jsdom#jsdom#whatwg-url#lodash"
info Disk size without dependencies: "4.88MB"
info Disk size with unique dependencies: "4.88MB"
info Disk size with transitive dependencies: "4.88MB"
info Number of shared dependencies: 0
✨  Done in 0.60s.
```

Reply:
reverted